dallinger/experiment_server/experiment_server.py

```diff
@@ -1244,6 +1246,34 @@ def api_notifications():
     return success_response(request_type="notification")
 
 
+@app.route("/experiment_status", methods=["GET"])
```

Comment:
I'll take a look at /summary and use that if it seems appropriate, otherwise I will rename as /experiment/status, and maybe remove the worker count info since there's already an API for that. I initially implemented it with just the completion boolean, but figured it might be nice to have a bit more information about experiment progress available for logging/debugging when running the experiment from python.